compose/cli/main.py

```diff
@@ -234,6 +234,7 @@ def ps(self, project, options):
 
         Options:
             -q    Only display IDs
+            --json    List containers in JSON format
```

Comment:
E.g.: docker-compose --json logs or up -d ? I think it would be the best.
If you agree update the PR's patches based on this advice.